data_structures/tree_234.cpp

```diff
@@ -0,0 +1,1278 @@
+/**
+ * @file
+ * @brief A demo 2-3-4 tree implementation
+ * @details Only implement some basic and complicated operations. Other
+ * operations should be easy to added.
+ * @author [liuhuan](https://github.com/fedom)
+ */
+#include <array>
+#include <cassert>
+#include <cstdio>
+#include <fstream>
+#include <iostream>
+#include <memory>
+#include <queue>
+#include <string>
```

Comment:
Add a brief description of what these headers/libraries do (what do they add).
```suggestion
#include <array>     /// for std::array
#include <cassert>   /// for assert
#include <cstdio>   /// for
#include <fstream>  /// for
#include <iostream>  /// for io operations
#include <memory>   /// for
#include <queue>   /// for
#include <string>  /// for
```

Reply:
Fix it. I have checked the guideline and seems these include comments suggestions are not there. Maybe you can add them there for us to follow :).